History/Timeline of Sembaran History.md

```diff
@@ -24,43 +23,60 @@ Over a series of start-and-start campaigns over 60 years, the Drankorian armies
 
-By the 1300s DR, the Sembaran monarchy commanded fealty, either directly, from newly minted barons, or indirectly, from lords sworn to the barons and dukes sworn to the crown, from most of the lowlands between the [[Teft]] and the [[Enst]]. For the next hundred years or so, the [[Green Sea]] is peaceful and prosperous. [[Sembara]] grows rich on the strength of its agricultural wealth, and its borders solidify: the [[Teft]] to the north, the [[Enst]] to the south, with the western border stretching roughly in a line from the source of the [[Enst]] through the [[Aine Hills]] to [[Braebein]], the highlands north of [[Duchy of Arnsbury|Arnsbury]].
+By the end of the 1300s, the Duke of Sembara was broadly acknowledged as feudal lord by the Dukes of Chemin, Seham, and Wistford as well as many of the barons along the south banks of the Enst, the Aine Hills, and the lands between the Wistel and the Enst as far west as the Cleenseau Wood. 
+
+%% The above section could use a bit more clarifying details, and there should be at least one major crisis that drives increasing support for the House of Entranca and the Dukes of Sembara, but the specifics are not something I feel like making up right now 
+
+The intention is that in 1400, there is broadly:
+* Two highland realms: Ardlas and Lavnoch, each with a fairly different character
+	* Ardlas is Mos Numena, although with strong kestavan influences than elsewhere in Sembara, and broadly culturally aligned with Sembara, speaks the same language, but with an independent monarch and a very different system of government. This was originally a Drankorian province
+	* Lavnoch is kestavan and largely speaks Zimkan, having never been a Drankorian province. Has close ties to dwarven kingdoms, and is culturally aligned with the rest of Zimkova
+* The larger mass of Zimkova, with a TBD culture, but definitely not a kingdom per se
+* The Duchy of Brovna, based around Urlich Pass, very culturally aligned with Sembara, but independent and not acknowledging the Sembaran overlordship, and growing rich on the trade links that are being established with Chardon. Possibly somewhat Swiss inspired, with many independent cantons/villages and a very weak Duke. This place doesn't have to be very well developed as it gets entirely destroyed in the Great War
+* Tyrwingha, an independent fey-tinged place ruled by an elected monarch and Ethlenn's guidance
+* The core Sembaran duchies
```

Comment:
Key part is here. Still need to think of what Zimkova is like, but I was thinking something like the rota of the Kiev Rus or Pictish kingship, but with a kestavo twist: The idea would be that the people are semi-nomadic, returning to complex ritual spaces at the end of a summer season  of wandering, which would universally be home to a kestavo, and the kings are basically the tenders of these spaces.